packages/enzyme-adapter-utils/src/createMountWrapper.jsx

```diff
@@ -55,7 +60,7 @@ export default function createMountWrapper(node, options = {}) {
     }
   }
   WrapperComponent.propTypes = {
-    Component: PropTypes.oneOfType([PropTypes.func, PropTypes.string]).isRequired,
+    Component: PropTypes.oneOfType([PropTypes.func, PropTypes.string, specialType]).isRequired,
```

Comment:
`PropTypes.element`?

Reply:
its not an element right it's a component class, e..g `element.type`

Reply:
`$$typeof` isn't in react 13 (which is why `react-is` doesn't work on it), so i'm not sure that the propType is correct.

Reply:
We could have multiple things here - but we might want to add a propType to each adapter for "a component" - that way react 0.13 can specify that the function has to be a class component, for example, and not an SFC.

Reply:
isValidElementType that we added would be the the right check I think. We could pass that in and use it in a custom proptype

Reply:
@jquense This may be helpful since I just figured this same prop type check out: https://github.com/facebook/prop-types/issues/200#issuecomment-409391696

(Eagerly awaiting this PR merge)